docs/how-to/customize-branding.mdx

```diff
@@ -9,18 +9,44 @@ keywords:
 pagination_next: null
 pagination_prev: null
 last_update: 
-   date: 06/01/2023
-   author: Patricia McPhee
+   date: 08/13/2023
+   author: Harry Guo
 draft: false
 doc_type: how-to
 displayed_sidebar: mainSidebar
 ---
 
-import { Alert, Col, Row } from 'antd';
+## Introduction
 
-<Row>
-  <Col span={6}>
-    <Alert message="To do" type="warning" />
-  </Col>
-</Row>
-<br />
\ No newline at end of file
+In the Beyond Identity Console, there are a couple of places where you can customize the branding to fit your organization's needs. 
+The branding configuration is **realm-specific** and we've broken it down to the sections below.
+
+### General
+
+These are general fields that will be displayed across your application(s) and email(s).
+
+Currently, there are three general fields:
+- **Organization Name**: your organization name.
+- **Logo**: visual logo that is on display.
+- **Support Link**: the support link that is displayed.
+
+By default, this values are prefilled with Beyond Identity name, logo, and support link.
+
+### Sign up/Login Experience Theme
```

Comment:
I see the button background and text color showing up for signup / passkey creation, however I don't see them having effect on login. The passkey choice page buttons remain white with black text.

Reply:
Thanks for bringing this up! Will investigate and raise any concerns.

Reply:
Okay, just confirmed that the passkey choice buttons page isn't affected by this. The only affected buttons are:

- OTP form
- Passkey creation
- Post passkey creation